server/query/user_screen.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../connection/pg");
+
+module.exports = {
+  async create({ screen_canvas_id, user_billboard_id, user_screen_name, description, config }) {
+    let query = "INSERT INTO user_screens (screen_canvas_id, user_billboard_id, user_screen_name, description, config) ";
+    query += `VALUES ('${screen_canvas_id}', '${user_billboard_id}', '${user_screen_name}', '${description}', '${config}')`;
```

Comment:
is it best practice to do JSON.stringify on the client side or the back-end?

Reply:
I'm not fully sure what the best practice would be but I'd imagine it would be the back end since the front end isn't the only place that will be sending data to a back end. Up to you, if you would like to do the stringify in the front end I can approve this now.

Reply:
why don't i do a check on the back-end: if the config is an object, then i stringify it. if its a string i assume its already stringified and leave it alone

Reply:
yup, that's way better than what I suggested.

Reply:
(i might have looked up a good way to fix this lol)